examples/IBM.Watson.Discovery.v2.Examples/DiscoveryTestData/exampleConfigurationData.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
This and the other Discovery v2 example data files don't seem to get used anywhere.

Reply:
Ah you're right - I must've copied the whole directory. I will remove.